docs/rules/require-explicit-slots.md

```diff
@@ -0,0 +1,71 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/require-explicit-slots
+description: require slots to be explicitly defined with defineSlots
+---
+
+# vue/require-explicit-slots
+
+> require slots to be explicitly defined with defineSlots
+
+- :exclamation: <badge text="This rule has not been released yet." vertical="middle" type="error"> ***This rule has not been released yet.*** </badge>
+
+## :book: Rule Details
+
+This rule enforces all slots used in the template to be defined once
+in the `script setup` block with the [`defineSlots`](https://vuejs.org/api/sfc-script-setup.html) macro.
```

Comment:
Can this rule be changed to also check the options API to check if the slots option is set?

https://vuejs.org/api/options-rendering.html#slots

Reply:
I'll look into adding this today, thank you!